package.json

```diff
@@ -2,20 +2,24 @@
   "name": "react-align",
   "version": "2.0.6",
   "author": "KaWaite",
-  "module": "dist/react-align.esm.js",
   "license": "MIT",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "./dist/react-align.umd.js",
+  "module": "./dist/react-align.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/react-align.mjs",
+      "require": "./dist/react-align.umd.js"
+    }
+  },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
-    "test": "tsdx test --passWithNoTests",
-    "lint": "tsdx lint",
-    "prepare": "tsdx build",
-    "size": "size-limit",
-    "analyze": "size-limit --why",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build": "tsc && vite build",
+    "type": "tsc",
+    "test": "vitest",
+    "lint": "eslint ."
+  },
+  "resolutions": {
+    "**/@types/react": "18.0.15"
   },
```

Comment:
Try refreshing yarn.lock (delete yarn.lock and run `yarn` again), and you might be able to delete resolutions.

Reply:
I just removed it and it seems to be fine...
For the record, this was solving a problem with incompatibility with react-beautiful-dnd and React 18, since dnd is written for older React versions and not maintained anymore. This was a suggested solution by someone on github and it worked. But now, for some reason it seems to stop being necessary..